src/LSODA.h

```diff
@@ -51,8 +51,8 @@ class LSODA
     int iopt; 
     int jt;
     int itask;
-    double Atol; 
-    double Rtol;
+    double Atol=1e-8; 
+    double Rtol=1e-8;
```

Comment:
will `LSODA` class be ever constructed with these default values? when will these values be in effect? Is that scenario in unit test?

Reply:
- Moved all initialization to the constructor
- Removed default constructor